app/components/primer/alpha/action_list.rb

```diff
@@ -84,8 +96,25 @@ def custom_element_name
           as: :item
         },
 
+        avatar_item: {
+          renders: lambda { |src:, username:, full_name: nil, full_name_scheme: Primer::Alpha::ActionList::Item::DEFAULT_DESCRIPTION_SCHEME, avatar_arguments: {}, **system_arguments|
+            build_item(label: username, description_scheme: full_name_scheme, **system_arguments).tap do |item|
+              item.with_leading_visual_raw_content do
+                # no alt text necessary for presentational item
+                render(Primer::Beta::Avatar.new(src: src, **avatar_arguments, role: :presentation, size: 16))
+              end
+
+              item.with_description_content(full_name) if full_name
+
+              will_add_item(item)
+            end
+          },
+
+          as: :avatar_item
+        },
+
         divider: {
-          renders: Divider,
+          renders: ActionList::Divider,
```

Comment:
🤘🏻

Reply:
I don't think this change was strictly necessary, but it's nice to be more explicit here since `NavList::Divider` is also a thing.

Reply:
I've seen some warnings because we didn't explicitly put the parent class